src/rules/require-event-dispatcher-types.ts

```diff
@@ -0,0 +1,28 @@
+import { createRule } from "../utils"
+import { extractCreateEventDispatcherReferences } from "./reference-helpers/svelte-createEventDispatcher"
+
+export default createRule("require-event-dispatcher-types", {
+  meta: {
+    docs: {
+      description: "require type parameters for createEventDispatcher",
+      category: "Best Practices",
+      recommended: false,
+    },
+    schema: [],
+    messages: {
+      missingTypeParameter: `Type parameters missing for the createEventDispatcher function call.`,
+    },
+    type: "suggestion",
+  },
+  create(context) {
+    return {
+      Program() {
```

Comment:
I think we need to check if `lang="ts"` is specified before we check it.

```suggestion
    let isTs = false
    return {
      SvelteScriptElement(node) {
        const lang = getLangValue(node)?.toLowerCase()
        if (lang === "ts" || lang === "typescript") {
          isTs = true
        }
      },
      "Program:exit"() {
        if (isTs) return
```